### PR TITLE
Do not fail hard when the conversation wasn't found (yet)

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -104,23 +117,9 @@ export default {
...
-		Router.beforeEach((to, from, next) => {
-			/**
-			 * This runs whenever the new route is a conversation.
-			 */
-			if (to.name === 'conversation') {
-				const NEXT_CONVERSATION_NAME = this.getConversationName(to.params.token)
-				this.setPageTitle(NEXT_CONVERSATION_NAME)
-			}
-			next()
-		})
```
Comment (ma12-co) [inline src/App.vue]: I think that we should keep the Router.beforeEach approach. 
Watchers are expensive and in the future we might need to trigger other things within `Router.beforeEach` ( even outside of `conversations` named routes ).

File: src/App.vue [hunk 2]
```diff
@@ -149,20 +148,6 @@ export default {
...
-		log(e) {
-			console.debug(e)
-		},
-		/**
-		 * Get a conversation's name.
-		 * @param {string} token The conversation's token
-		 * @returns {string} The conversation's name
-		 */
-		getConversationName(token) {
-			return this.$store.getters.conversations[token].displayName
```
Comment (nickvergessen) [inline src/App.vue]: Then we should at least block failuring here when the item is not in the list atm (for whatever reason)

Comment (ma12-co) [reply src/App.vue]: yes, definetly :)

**Comment by nickvergessen** [comment]
Fixed

**Pull request merged**